### PR TITLE
Do not fetch windows if remaining data points is <= 0

### PR DESCRIPTION
**Pull request by simenl**
## Description
Optimization:
Do not divide into windows if there are 0 remaining data points. Clients asking for 100k data points (the limit per request) will do one request instead of two.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change. 
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring. 
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by spex66 — APPROVED** [review]
lgtm based on our slack discussion

File: cognite/client/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-__version__ = "2.26.0"
+__version__ = "2.27.0"
```
Comment (me-ydv-5) [inline cognite/client/_version.py:1]:
```suggestion
__version__ = "2.26.1"
```
Think this more of a patch than minor fix?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.27.0] - 2021-07-20
```
Comment (me-ydv-5) [inline CHANGELOG.md:17]:
```suggestion
## [2.26.1] - 2021-07-20
```

**Review by me-ydv-5 — APPROVED** [review]

**Comment by me-ydv-5** [comment]
thread https://cognitedata.slack.com/archives/CG10VQPFX/p1626777498014100

**Pull request merged**